charts/kong/values.yaml

```diff
@@ -576,6 +576,8 @@ ingressController:
     service:
       # Specify custom labels for the validation webhook service.
       labels: {}
+    # Tune the default Kubernetes timeoutSeconds of 10 seconds
+    # timeoutSeconds: 10
```

Comment:
If we mention the default is 10 in the docs , why not set it here to 10?

Reply:
Because that default comes from `kube-apiserver` and not from Kong

Reply:
I think that we shouldn't override the Kubernetes default unless the user intentionally does that.